CONTRIBUTING.md

```diff
@@ -157,8 +165,10 @@ Note:
 - Only members of the core dev team with quite a bit of experience writing and reviewing PRs should merge pull requests. If you're unsure, ask first.
 - Only merge PRs with 2 or more concurrent approvals from members of the core dev team with limited exceptions.
 - Only merge PRs after the CI has passed and if there are no conflicts. Ideally rebase or merge with master first.
+- We prefer to have all feedback resolved before a PR is merged.
 - If you wrote it, don't merge it. It's best practice for someone else to decide it's ready to be merged.
 - To keep the commit history simple and allow for easy manipulation of PRs, we use the `Squash-and-Merge` functionality of GitHub's web interface.
+- Before merging, post a message in slack asking if anyone has objections to the PR being merged and wait at least 5 mins for others to reply. After merging, reply in thread to your original post to let the team know the PR has been merged.
```

Comment:
@Bears-R-Us/arkouda-core-dev this was just because people are not resolving all of their conversations, if we can be more diligent about that, we can remove/change this...